### PR TITLE
Exclude tests for remote libvirt clusters with multi-architecture compute nodes

### PR DESCRIPTION
**Pull request by werled**
This implements exclusion of tests for clusters on s390x + x86 remote libvirt clusters installed via UPI (https://issues.redhat.com/browse/MULTIARCH-4213)

The test exclusions used in ref `openshift-e2e-libvirt-test` are explicitly listed in `openshift-e2e-libvirt-conf`. Since we are reusing the former for clusters with multi-architecture compute nodes, also explictly listed all skips. This is in contrast to the regular expression used for skipping tests in `openshift-e2e-test`, which is used by other multi-architecture compute workflows.

### CODE REVIEW

**Comment by werled** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-remote-libvirt-s390x-heterogeneous

**Comment by werled** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-s390x-heterogeneous

**Comment by werled** [comment]
There was already a successful rehearsal for the 4.15 job before the 4.16 job PR merged and I rebased:
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/49427/rehearse-49427-periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-remote-libvirt-s390x-heterogeneous/1763537501919645696

File: ci-operator/step-registry/openshift/e2e/libvirt/conf/openshift-e2e-libvirt-conf-commands.sh [hunk 1]
```diff
@@ -323,6 +323,130 @@ EOF
...
+#        with buildah\| oc new-app should succeed\| prune builds based on settings\|
+#        s2i build with a root\| verify /run filesystem contents\| oc can run\| oc
+#        debug\| oc idle\| Pods cannot access\| Image append should create\| Image
+#        extract should extract\| Image info should display\| Image layer subresource\|
+#        oc tag should change image\| when installed on the cluster should\| OpenShift
+#        alerting rules\| The HAProxy router should\| egressrouter cni resources\|
+#        pod should start\| pod sysctls\| build volumes should mount given secrets
+#        and configmaps into the build pod
+HETEROGENEOUS_CLUSTER_TYPE_REGEX="^(libvirt-s390x-amd64)|(libvirt-amd64-s390x)$"
+if [ "${TEST_TYPE}" == "conformance-parallel" ] && [[ "${CLUSTER_TYPE}" =~ $HETEROGENEOUS_CLUSTER_TYPE_REGEX ]]; then
```
Comment (tvardema) [inline ci-operator/step-registry/openshift/e2e/libvirt/conf/openshift-e2e-libvirt-conf-commands.sh:343]: Nitpick: can we maintain variable reference consistency here?  Thanks!
`${HETEROGENEOUS_CLUSTER_TYPE_REGEX}`

Comment (werled) [reply ci-operator/step-registry/openshift/e2e/libvirt/conf/openshift-e2e-libvirt-conf-commands.sh]: Done. Good that you pointed me to that place again, regex was also slightly wrong (`"^(libvirt-s390x-amd64)|(libvirt-amd64-s390x)$"` should be `"^(libvirt-s390x-amd64|libvirt-amd64-s390x)$"`).

**Comment by werled** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-remote-libvirt-s390x-heterogeneous

**Comment by werled** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-remote-libvirt-s390x-heterogeneous

**Comment by werled** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-remote-libvirt-s390x-heterogeneous

**Comment by werled** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-s390x-heterogeneous

**Comment by werled** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-s390x-heterogeneous

**Pull request merged**